nixos/modules/system/boot/systemd-nspawn.nix

```diff
@@ -42,8 +42,127 @@ let
     (assertValueOneOf "VirtualEthernet" boolValues)
   ];
 
+  bindMounts = mkOption {
+    type = with types; attrsOf (submodule {
+      options = {
+        source = mkOption {
+          type = either str path;
+          default = "";
+          description = "Optional mount source path. If omitted, the specified path will be mounted from the host to the same path in the container.";
+        };
+        mountOption = mkOption {
+          type = str;
+          default = "";
+          description = "Optional mount options. See https://www.freedesktop.org/software/systemd/man/systemd-nspawn.html#Mount%20Options for more info.";
```

Comment:
(nitpick) You may want to use `<link xlink:href="url">linktext</link>` here. It will be properly rendered in the man-page and the link will be clickable in the HTML-output of the manual.

Reply:
Thanks, fixed.

BTH, any documentation of the xml?